proposals/OBv3p0/specification.md

```diff
@@ -0,0 +1,374 @@
+
+* _Add recipient to the credentialSubject.id_: VCs have credentialSubject which references the claim being being verified. The credentialSubject ID property can reference the subject of the credential. This property can contain a URI representing the subject which may be a URL or a DID. It should be discussed whether other profile properties could be optionally included in the credentialSubject.
+
+* _Issuer & Credential Subject Identities_: With VCs, [issuer](https://www.w3.org/TR/vc-data-model/#issuer) and [credentialSubject](https://www.w3.org/TR/vc-data-model/#credential-subject) id properties must be a URI. This could be an HTTP-based URL or a [Decentralized Identifier](https://www.w3.org/TR/did-use-cases/#intro).
+
+* _credentialSubject.hasCredential_: The hasCredential property from schema.org is intended to contain the BadgeClass properties including name, description, criteria, etc. hasCredential.id should contain the canonical url for the BadgeClass. hasCredential.type identifies the properties as belonging to the BadgeClass.
+
+* _credentialSubject.hasCredential.achievementType_: This property is taken from the CLR and may contain the same string values as proposed by the CLR.
+
+* _credentialSubject.hasCredential.creator_: Open Badges as native VCs should continue to be one issuer of one claim about one recipient. With Open Badges, the issuer profile has been referenced in the BadgeClass. With Open Badges as VCs, the issuer is the entity that signs the credential. The issuer may be different from the creator of the badge being issued. As depicted in the example below, with Open Badges as VCs, the issuer profile may still include name, description, url, image, email, etc.
+
+* _credentialSubject.evidence_: With OpenBadges up to 2.0, the evidence has been an assertion property. This proposal suggests that evidence should be included in the credentialSubject object so that the evidence is related to the claim, not the verification of the claim.
+
+* _credentialSubject.resultDescription_: This property is taken from the CLR and may follow the same recommendations. As with evidence, this property should be included in the credentialSubject object.
```

Comment:
Pulling in the _resultDescriptions_ from CLR Achievements is probably not very useful without also pulling in the _results_ from CLR Assertions.

I would also think the field would make more sense under the _hasCredential_ as it is part of the definition of a credential/achievement/BadgeClass and not related to the subject.

Reply:
@roverwolf Yes this is correct. _credentialSubject.result_ would be the right field to reference here. `resultDescription` should be in the Achievement/BadgeClass class.